ambry-commons/src/main/java/com.github.ambry.commons/ServerErrorCode.java

```diff
@@ -27,5 +27,7 @@
   Partition_Unknown,
   Disk_Unavailable,
   Partition_ReadOnly,
-  Unknown_Error
+  Unknown_Error,
```

Comment:
These enums unfortunately cannot be sorted (also `RequestOrResponseType`). They are used in serialization and deserialization and their relative order is important since the ordinal (their position in the list) is what goes into the serialized form.